spiel/st/m0t1fs_spiel_st.sh

```diff
@@ -545,7 +545,7 @@ _mount() {
     local MOUNT_OPTS="-t m0t1fs -o pfid=<0x7200000000000001:1>,profile=$PROF_OPT,ha=$M0D2_ENDPOINT \
 none $SANDBOX_DIR/mnt"
     echo "mount $MOUNT_OPTS"
-    mount $MOUNT_OPTS || return $?
+    mount "$MOUNT_OPTS" || return $?
```

Comment:
MOUNT_OPTS should not be in double quotes

Reply:
incorporated